commit fb3be2841c107448d1201c62f3fb8cabab229066
Author: Marc Haber <mh+debian-bugs@zugschlus.de>
Date:   Tue Feb 7 10:52:31 2017 +0100

    NMU: Prevent ENGINE_by_id failed in chroot by disabling GOST
    
    This annoyance seems to prevent bind9 from running in a chroot, which is one
    of the most commons patterns of deployment.
    
    Closes: #820974
    Signed-off-by: Marc Haber <mh+debian-bugs@zugschlus.de>
    Signed-off-by: Arturo Borrero Gonzalez <arturo@debian.org>

diff --git a/debian/changelog b/debian/changelog
index 0601dc24c..692756e84 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+bind9 (1:9.10.3.dfsg.P4-11.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Disable GOST to prevent ENGINE_by_id failed (crypto failure) in chroot.
+    Patch by Marc Haber <mh+debian-bugs@zugschlus.de> (Closes: #820974).
+
+ -- Arturo Borrero Gonzalez <arturo@debian.org>  Tue, 07 Feb 2017 10:42:00 +0100
+
 bind9 (1:9.10.3.dfsg.P4-11) unstable; urgency=medium
 
   * Fix some lintian warnings.
diff --git a/debian/rules b/debian/rules
index 80a33383b..e26361828 100755
--- a/debian/rules
+++ b/debian/rules
@@ -61,6 +61,7 @@ stamps/configure: stamps/prepare
 		--with-libtool \
 		--enable-shared \
 		--enable-static \
+		--with-gost=no \
 		--with-openssl=/usr \
 		--with-gssapi=/usr \
 		--with-gnu-ld \
